src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResource.kt

```diff
@@ -0,0 +1,105 @@
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.data.CodeDescription
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@RestController
+@Validated
+@RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
+class SentencingResource(private val sentencingService: SentencingService) {
+  @PreAuthorize("hasRole('ROLE_NOMIS_SENTENCING')")
+  @GetMapping("/prisoners/{offenderNo}/sentencing/court-case/{id}")
+  @Operation(
+    summary = "get a court case",
+    description = "Requires role NOMIS_SENTENCING. Retrieves a court case by id",
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "the court case details",
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden to access this endpoint when role NOMIS_SENTENCING not present",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Court case not found",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Offender not found",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+    ],
+  )
+  fun getCourtCase(
+    @Schema(description = "Court case id", example = "12345")
+    @PathVariable
+    id: Long,
+    @Schema(description = "Offender No", example = "12345")
+    @PathVariable
+    offenderNo: String,
+  ): CourtCaseResponse = sentencingService.getCourtCase(id, offenderNo)
+}
+
+@Schema(description = "Court Case")
+data class CourtCaseResponse(
+  val id: Long,
+  val offenderNo: String,
+  val caseInfoNumber: String?,
+  val caseSequence: Int,
+  val caseStatus: CodeDescription,
+  val caseType: CodeDescription,
+  val beginDate: LocalDate?,
+  val prisonId: String,
+  val combinedCaseId: Long?,
+  val statusUpdateStaffId: Long?,
+  val statusUpdateDate: LocalDate?,
+  val statusUpdateComment: String?,
+  val statusUpdateReason: String?,
+  val lidsCaseId: Int?,
+  val lidsCombinedCaseId: Int?,
+  val lidsCaseNumber: Int,
+  val createdDateTime: LocalDateTime,
+  val createdByUsername: String,
```

Comment:
the majority of field comments don't add value here - tempted to leave off for this internal api.